app/editor/src/features/content/form/ContentClipForm.tsx

```diff
@@ -48,6 +48,8 @@ export const ContentClipForm: React.FC<IContentClipFormProps> = ({
   const [item, setItem] = React.useState<IItemModel>();
   const [start, setStart] = React.useState<string>('');
   const [end, setEnd] = React.useState<string>('');
+  const [startPoint, setStartPoint] = React.useState<string>('');
```

Comment:
We don't need additional state for display purposes.  We already have the `start` and `end` values, we just use the functions to display them appropriately.

Reply:
Updated, thanks